### ISSUE TITLE
Unable to install net6.0

### ISSUE THREAD

**Issue by Kaotic3** [dotnet/BenchmarkDotNet#2105]
I have installed this before no issue, so I am not sure why I am having an error now (to clarify, on different machines, not this one) - judging by there being a lack of posts, more likely my end than yours, but thought I would check.

As a double check, I added a different Nuget Package that I often use, AhoCorasick - that installed no issue.  I also spooled up another Console application - entirely separate from the first, same issue.

When I attempt to add BenchmarkDotNet via the Manage Nuget Packages in Visual Studio Community 2022, I get the following error messages - for the record, I rolled back from 0.13.2 to 0.13.1 - but got the same error.

![image](https://user-images.githubusercontent.com/80522225/190506912-31bd66c9-0d3d-46d7-86ea-6ff342830e61.png)

This was a base console application, with nothing in it at all - twice.  I haven't even written any code.

This is the csproj:

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net6.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
  </PropertyGroup>

</Project>
```

I looked at the troubleshooting page - but that sort of requires that it is installed and running but having issues.

I did use the getting started and tried to install via the Powershell command:

`Install-Package BenchmarkDotNet`

It has been sat there for about 15 minutes while I wrote this doing nothing, so I gave up on it.

Any chance this has been seen before?

**Comment by Kaotic3** [comment]
I uninstalled Visual Studio completely and reinstalled it - in the hope that this was a VS problem.

Same error messages.

**Comment by AndreyAkinshin** [comment]
@Kaotic3 could you please share the output of the following commands (run them from the project root):
```
dotnet --list-sdks
dotnet --list-runtimes
dotnet add package BenchmarkDotNet
```
Also, please share the content of your `%appdata%\NuGet\NuGet.Config` file.

**Comment by Kaotic3** [comment]
dotnet --list-sdks:

```
6.0.300 [C:\Program Files\dotnet\sdk]
6.0.401 [C:\Program Files\dotnet\sdk]
```

dotnet --list-runtimes:

```
Microsoft.AspNetCore.App 6.0.5 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.AspNetCore.App 6.0.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.NETCore.App 3.1.8 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 6.0.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 6.0.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.WindowsDesktop.App 3.1.8 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
Microsoft.WindowsDesktop.App 6.0.5 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
Microsoft.WindowsDesktop.App 6.0.9 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

dotnet add package BenchmarkDotNet:

```
Writing C:\Users\Kaos\AppData\Local\Temp\tmp2C14.tmp
info : X.509 certificate chain validation will use the default trust store selected by .NET.
info : Adding PackageReference for package 'BenchmarkDotNet' into project 'C:\Users\Kaos\source\repos\BenchMarkTestingv23\BenchMarkTestingv23\BenchMarkTestingv23.csproj'.
info :   CACHE https://api.nuget.org/v3/registration5-gz-semver2/benchmarkdotnet/index.json
info : Restoring packages for C:\Users\Kaos\source\repos\BenchMarkTestingv23\BenchMarkTestingv23\BenchMarkTestingv23.csproj...
error: NU1202: Package BenchmarkDotNet 0.13.2 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package BenchmarkDotNet 0.13.2 does not support any target frameworks.
error: NU1202: Package BenchmarkDotNet.Annotations 0.13.2 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package BenchmarkDotNet.Annotations 0.13.2 does not support any target frameworks.
error: NU1202: Package CommandLineParser 2.4.3 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package CommandLineParser 2.4.3 does not support any target frameworks.
error: NU1202: Package Iced 1.17.0 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Iced 1.17.0 does not support any target frameworks.
error: NU1202: Package Microsoft.CodeAnalysis.CSharp 3.0.0 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.CodeAnalysis.CSharp 3.0.0 does not support any target frameworks.
error: NU1202: Package Microsoft.Diagnostics.Runtime 2.2.332302 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.Diagnostics.Runtime 2.2.332302 does not support any target frameworks.
error: NU1202: Package Microsoft.DotNet.PlatformAbstractions 3.1.6 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.DotNet.PlatformAbstractions 3.1.6 does not support any target frameworks.
error: NU1202: Package Microsoft.CodeAnalysis.Common 3.0.0 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.CodeAnalysis.Common 3.0.0 does not support any target frameworks.
error: NU1202: Package Microsoft.Diagnostics.NETCore.Client 0.2.251802 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.Diagnostics.NETCore.Client 0.2.251802 does not support any target frameworks.
error: NU1202: Package System.Text.Encoding.CodePages 4.5.0 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package System.Text.Encoding.CodePages 4.5.0 supports:
error:   - monoandroid10 (MonoAndroid,Version=v1.0)
error:   - monotouch10 (MonoTouch,Version=v1.0)
error:   - xamarinios10 (Xamarin.iOS,Version=v1.0)
error:   - xamarinmac20 (Xamarin.Mac,Version=v2.0)
error:   - xamarintvos10 (Xamarin.TVOS,Version=v1.0)
error:   - xamarinwatchos10 (Xamarin.WatchOS,Version=v1.0)
error: NU1202: Package Microsoft.Extensions.Logging 2.1.1 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.Extensions.Logging 2.1.1 does not support any target frameworks.
error: NU1202: Package Microsoft.Extensions.Logging.Abstractions 2.1.1 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.Extensions.Logging.Abstractions 2.1.1 does not support any target frameworks.
error: NU1202: Package Microsoft.Extensions.Configuration.Binder 2.1.1 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.Extensions.Configuration.Binder 2.1.1 does not support any target frameworks.
error: NU1202: Package Microsoft.Extensions.DependencyInjection.Abstractions 2.1.1 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.Extensions.DependencyInjection.Abstractions 2.1.1 does not support any target frameworks.
error: NU1202: Package Microsoft.Extensions.Options 2.1.1 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.Extensions.Options 2.1.1 does not support any target frameworks.
error: NU1202: Package Microsoft.Extensions.Configuration 2.1.1 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.Extensions.Configuration 2.1.1 does not support any target frameworks.
error: NU1202: Package Microsoft.Extensions.Primitives 2.1.1 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.Extensions.Primitives 2.1.1 does not support any target frameworks.
error: NU1202: Package Microsoft.Extensions.Configuration.Abstractions 2.1.1 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package Microsoft.Extensions.Configuration.Abstractions 2.1.1 does not support any target frameworks.
error: Package 'BenchmarkDotNet' is incompatible with 'all' frameworks in project 'C:\Users\Kaos\source\repos\BenchMarkTestingv23\BenchMarkTestingv23\BenchMarkTestingv23.csproj'.
```

**Comment by Kaotic3** [comment]
If it is of any value, I realised that I had actually run BenchmarkDotNet on this machine (I thought I had only used it on the work machine).

I opened up the project containing Benchmark and attempted to run it and received these two error messages:

`Error MSB3030 Could not copy the file "C:\Users\Kaos\.nuget\packages\commandlineparser\2.4.3\lib\netstandard2.0\CommandLine.dll" because it was not found.  RegexTesting  C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets 4809  `

Looking in the folder location, there is no .dll file - there is a xml file, but it is 2,000 lines of what appears to be various exception messages.

`Error MSB3030 Could not copy the file "C:\Users\Kaos\.nuget\packages\perfolizer\0.2.1\lib\netstandard2.0\Perfolizer.dll" because it was not found.  RegexTesting  C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets 4809  `

There is no \lib folder, and therefore no \netstandard2.0 folder in this location at all.
